src/LibraryManager.Vsix/Json/Completion/SimpleCompletionEntry.cs

```diff
@@ -24,6 +24,11 @@ public SimpleCompletionEntry(string text, ImageMoniker moniker, IIntellisenseSes
         {
         }
 
+        public SimpleCompletionEntry(string text, string insertionText, ImageMoniker moniker, IIntellisenseSession session)
```

Comment:
name this `displayText`, as that's how we distinguish in the completion APIs

Reply:
We should fixup the other methods where we have separate display/insertion text parameters as well.

---
In reply to: [196958553](https://github.com/aspnet/LibraryManager/pull/184#discussion_r196958553) [](ancestors = 196958553)